api/tests/routes/native/v1/account_test.py

```diff
@@ -868,6 +868,107 @@ def test_ongoing_email_update_blocks_the_next_update_request(self, client):
         assert second_response.json["error_code"] == account_errors.EMAIL_UPDATE_PENDING.code
         assert second_response.json["message"] == account_errors.EMAIL_UPDATE_PENDING.message
 
+    def test_end_to_end_update_email(self, client):
+        password = "some_random_string"
+        old_email = self.identifier
+        new_email = "updated_" + old_email
+        user = users_factories.UserFactory(email=old_email, password=password)
+        client.with_token(user.email)
+        response = client.post(
```

Comment:
(suggestion / question de goût) Je trouve que les tests d'intégration sont aussi éducatifs et ici il semblerait utile de parsemer quelques commentaires courts pour donner les grandes étapes du parcours : 

- "User asks to change change their e-mail address."
- "User receives an e-mail on their _current_ e-mail address, asking to follow a link to confirm."
- "User confirms (from the link sent to their _current_ e-mail address)."
- "User receives an e-mail on their _new_ e-mail address, asking to follow a link to confirm."
- "User confirms (from the link sent to their _new_ e-mail address)."